be/src/storage/rowset/array_column_iterator.cpp

```diff
@@ -20,15 +20,17 @@
 
 namespace starrocks {
 
-ArrayColumnIterator::ArrayColumnIterator(std::unique_ptr<ColumnIterator> null_iterator,
+ArrayColumnIterator::ArrayColumnIterator(ColumnReader* reader, std::unique_ptr<ColumnIterator> null_iterator,
```

Comment:
const?

Reply:
it's will assign to class's private parameter, i think don't need const?